Test/comprehensive_performance_test.py

```
@@ -0,0 +1,1115 @@
+        self.should_stop = True
+        time.sleep(1)
+        
+        # 결과 리포트
+        self.report_results()
+        
+        # 연결 종료
+        if self.socket:
+            try:
+                self.socket.close()
+            except:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)